backend/package.json

```diff
@@ -15,18 +15,23 @@
   "author": "",
   "license": "ISC",
   "dependencies": {
+    "cloudinary": "^1.25.2",
     "cors": "^2.8.5",
+    "datauri": "^3.0.0",
     "dotenv": "^8.2.0",
     "express": "^4.17.1",
     "joi": "^17.4.0",
     "mongodb-memory-server": "^6.9.6",
-    "mongoose": "^5.12.1"
+    "mongoose": "^5.12.1",
+    "node-fetch": "^2.6.1",
```

Comment:
We are using axios to any requests (backend and frontend). Please update this package to proper one. Not a warning or error, but in axios its easier to handle response and errors